BREAKING_CHANGES_FOR_V15.md

```diff
@@ -0,0 +1,41 @@
+# Breaking change notice for version 15.0.0
+
+## Removal of `ShopifyAPI::Webhooks::Handler`
+
+The `ShopifyAPI::Webhooks::Handler` class has been removed in favor of `ShopifyAPI::Webhooks::WebhookHandler`. The `ShopifyAPI::Webhooks::WebhookHandler` class is now the recommended way to handle webhooks.
+
+Make a module or class that includes or extends `ShopifyAPI::Webhooks::WebhookHandler` and implement the `handle` method which accepts the following named parameters: data: `WebhookMetadata`.
```

Comment:
Should we add a tiny sentence here explaining why we did this, like:

```
In v14, adding new fields to the callback would become a breaking change. To make this code more flexible, handlers will now receive an object that can be typed and extended.
```

Reply:
I like that! Clarity on why we are making them do this migration.